teps/0025-proxy-support.md

```diff
@@ -0,0 +1,133 @@
+---
+title: proxy-support
+
+authors:
+  - "@piyush-garg"
+  - "@vdemeester"
+  
+creation-date: 2020-03-14
+
+last-updated: 2020-03-14
+
+status: proposed
+
+---
+
+# TEP-0025: Proxy Support
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (optional)](#user-stories-optional)
+  - [Notes/Constraints/Caveats (optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+  - [User Experience (optional)](#user-experience-optional)
+  - [Performance (optional)](#performance-optional)
+- [Design Details](#design-details)
+- [Test Plan](#test-plan)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (optional)](#infrastructure-needed-optional)
+- [Upgrade &amp; Migration Strategy (optional)](#upgrade--migration-strategy-optional)
+- [References (optional)](#references-optional)
+<!-- /toc -->
+
+## Summary
+
+The proposal is to provide the proxy support to tektoncd/pipelines and tektoncd/triggers.
+
+Tekton should provide a way to use network proxy settings defined for the platform so that workloads
+can make use of it during execution.
+
+## Motivation
+
+This will enable users/organisations to enable network proxy settings for CI/CD using tekton if their
```

Comment:
This should be something like :
"Support users and organisations that require a network proxy for their workload, including CI/CD"